content/Watsonx/CodeAssistant/README.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'IBM Watsonx Code Assistant Technical Sales Workshop'
+toc: false
+---
+
+# IBM Watsonx Code Assistant Technical Sales Workshop
+
+This workshop covers the IBM Watsonx Code Assistant L3 lab, highlighting some of the core components and capabilities of IBM Watsonx Code Assistant. 
+
+## Prerequisites
+
+Before proceeding through this lab, make sure you have installed and have access to the following:
+
+- [Visual Studio Code](https://code.visualstudio.com/download) (VS Code)
+- [Python](https://www.python.org/downloads/) 3.9+: VSCode uses Python to execute Ansible playbooks and roles.
+- [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-and-upgrading-ansible-with-pip): You can install Ansible with the below command in your Terminal or Command Line.
+    ```bash
+    pip install ansible
```

Comment:
the pip command for me pointed to a system version of python that was 2.7.  I needed pip3 install ansible (or I needed to add an alias pip=pip3.  In case others have a similar situation, should we just point at the install guide and leave out the bash command?

Reply:
okay, I removed the bash command.